xml/System.Net.WebSockets/WebSocket.xml

```diff
@@ -400,12 +400,12 @@
         <Parameter Name="keepAliveInterval" Type="System.TimeSpan" Index="3" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <param name="isServer">To be added.</param>
-        <param name="subProtocol">To be added.</param>
-        <param name="keepAliveInterval">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="stream">The <see cref="T:System.IO.Stream" /> for the connection.</param>
+        <param name="isServer"><see langword="true" /> if this is the server-side of the connection; <see langword="false" /> if it's the client side.</param>
+        <param name="subProtocol">The agreed upon sub-protocol that was used when creating the connection.</param>
+        <param name="keepAliveInterval">The keep-alive interval to use, or <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" /> to disable keep-alives.</param>
+        <summary>Creates a <see cref="T:System.Net.WebSockets.WebSocket" /> that operates on a <see cref="T:System.IO.Stream" /> representing a web socket connection.</summary>
+        <returns>The created <see cref="T:System.Net.WebSockets.WebSocket" />.</returns>
         <remarks>To be added.</remarks>
```

Comment:
Remarks are not needed.
```suggestion
```

Reply:
Those elements were autogenerated by the Docs team, I don't think we should remove them. The "To be added" string is what they consider "not documented". Since remarks are optional, they can stay with that value. They won't show up among the undoc APIs in our stats.